example/src/main/tut/ct/Kleisli.md

```diff
@@ -3,12 +3,11 @@ layout: docs
 title:  "Kleisli"
 ---
 
-# Kleisli
+# Kleisli [![GitHub](../img/github.png)](https://github.com/scalaz/scalaz/blob/series/8.0.x/base/shared/src/main/scala/scalaz/ct/kleisli.scala)
```

Comment:
All the hard coded links look hard to maintain. Is there a better way?

Reply:
Unfortunately, no. The branch name should never change? So only larger refactors should have any impact on this.